.github/workflows/code-coverage.yml

```diff
@@ -0,0 +1,185 @@
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+      - master
+      - devel
+  pull_request:
+    branches:
+      - main
+      - master
+      - devel
+
+name: Code Coverage ☂
+
+jobs:
+  coverage:
+    name: Test Coverage 🧪
+    runs-on: ubuntu-20.04
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+    outputs:
+      coverage-percent: ${{ steps.set-coverage-percentage.outputs.coverage-percentage }}
+    steps:
+      - uses: actions/cache@v2
+        with:
+          path: ~/.local/share/renv
+          key: ${{ runner.os }}-renv-${{ hashFiles('**/renv.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-renv-
+
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Setup R
+        uses: r-lib/actions/setup-r@v1
+        with:
+          r-version: 3.6
+
+      - name: Setup Pandoc
+        uses: r-lib/actions/setup-pandoc@v1
+
+      - name: Query dependencies
+        run: |
+          install.packages('remotes')
+          saveRDS(remotes::dev_package_deps(dependencies = TRUE), ".github/depends.Rds", version = 2)
+          writeLines(sprintf("R-%i.%i", getRversion()$major, getRversion()$minor), ".github/R-version")
+        shell: Rscript {0}
+
+      - name: Install system dependencies
+        run: |
+          while read -r cmd
+          do
+            eval sudo $cmd
+          done < <(Rscript -e 'writeLines(remotes::system_requirements("ubuntu", "20.04"))')
+
+      - name: Install dependencies
+        run: |
+          if (!requireNamespace("renv", quietly = TRUE)) install.packages("renv")
+          renv::restore()
+        shell: Rscript {0}
+
+      - name: Install package
+        run: R CMD INSTALL .
+
+      - name: Run coverage 👟
+        run: |
+          tryCatch(
+            expr = {
+                x <- covr::package_coverage(
+                  path=".",
+                  clean = FALSE,
+                  quiet = FALSE
+                )
+                print(x)
+                covr::to_cobertura(x, filename = "coverage.xml")
```

Comment:
I was wondering how to convert the report to xml.  this is really cool!

Reply:
Yeah this is definitely one of the nice things about the package. We can export to different formats.